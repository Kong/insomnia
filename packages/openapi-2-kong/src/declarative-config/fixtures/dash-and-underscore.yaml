openapi: 3.0.0
info:
  version: 1.0.0
  title: Sample-API
servers:
  - url: https://example.com
paths:
  /list:
    get:
      description: Returns a list of stuff
      responses:
        '200':
          description: Successful response
