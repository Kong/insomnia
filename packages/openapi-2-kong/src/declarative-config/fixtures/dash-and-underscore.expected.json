{
    "_format_version": "1.1",
    "services": [
        {
            "name": "Sample-API",
            "protocol": "https",
            "host": "example.com",
            "port": 443,
            "path": "/",
            "plugins": [],
            "routes": [
                {
                    "tags": [
                        "OAS3_import",
                        "OAS3file_dash-and-underscore.yaml"
                    ],
                    "name": "Sample-API-list-get",
                    "methods": [
                        "GET"
                    ],
                    "paths": [
                        "/list$"
                    ],
                    "strip_path": false
                }
            ],
            "tags": [
                "OAS3_import",
                "OAS3file_dash-and-underscore.yaml"
            ]
        }
    ]
}
